spec/system/custom/polls/polls_spec.rb

```diff
@@ -18,11 +18,27 @@
       expect(page).to have_css(".unverified", count: 3)
       expect(page).to have_content("You must verify your account to participate")
     end
+
+    scenario "Already participated in a poll" do
+      question = create(:poll_question, :yes_no)
+      login_as(create(:user, :level_two))
+
+      visit poll_path(question.poll)
+      choose "Yes"
+      click_button "Send"
+
+      expect(page).to have_content("Poll saved successfully!")
+
+      visit polls_path(question.poll)
```

Comment:
Is there a typo here? Did we mean:

```suggestion
      visit polls_path
```